public/locales/en/loginBtn.json

```
@@ -0,0 +1,4 @@
+{
+    "Sign in": "تسجيل الدخول",
+    "Sign out": "تسجيل الخروج"
```

this file is English file, so the values should be in English, right?
```suggestion
    "Sign in": "Sign in",
    "Sign out": "Sign out"
```

---

Done